detekt.yml

```
@@ -0,0 +1,10 @@
+style:
+  MaxLineLength:
+    maxLineLength: 100
+  MagicNumber:
+    excludes:
+      - '**/test/**'
+      - '**/commonTest/**'
+      - '**/jvmTest/**'
+      - '**/examples/**'
+
```

YAMLlint (1.38.0)

[error] too many blank lines (1 > 0)

(empty-lines)